esp-hal/src/gpio/any_pin.rs

```diff
@@ -1,39 +1,25 @@
 use super::*;
 
-#[derive(Clone, Copy)]
-enum Inverted {
-    NonInverted,
-    Inverted,
-}
-
-impl Inverted {
-    fn is_inverted(&self) -> bool {
-        match self {
-            Inverted::NonInverted => false,
-            Inverted::Inverted => true,
-        }
-    }
-}
-
-/// Generic pin wrapper for pins which can be Output or Input.
+/// A type-erased GPIO pin, with additional configuration options.
+///
+/// Note that accessing unsupported pin functions (e.g. trying to use an
+/// input-only pin as output) will panic.
 pub struct AnyPin<'d> {
     pin: ErasedPin,
-    inverted: Inverted,
+    inverted: bool,
```

Comment:
New type, new name. 😛
```suggestion
    is_inverted: bool,
```

Reply:
Fair enough.